Octokit.Tests.Integration/fixtures/OrganizationsHooksFixture.cs

```diff
@@ -0,0 +1,42 @@
+﻿using System;
+using System.Collections.Generic;
+
+namespace Octokit.Tests.Integration.fixtures
+{
+    public class OrganizationsHooksFixture : IDisposable
+    {
+        readonly IGitHubClient _github;
+        readonly OrganizationHook _hook;
+        private string organizationFixture;
```

Comment:
local class variable should be named `_organizationFixture` and be `readonly` if possible